<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-M4HP65I" failures="3" tests="23" name="Regression suite 1" time="0.279" errors="0" timestamp="2023-08-18T14:30:59 IST">
  <testcase classname="testNGDemo.NewTest" name="f" time="0.006"/>
  <testcase classname="testNGDemo.TestngAnnotationDemo1" name="funA" time="0.002"/>
  <testcase classname="testNGDemo.TestngAnnotationDemo1" name="funB" time="0.003"/>
  <testcase classname="testNGDemo.TestngAnnotationsDemo2" name="funA" time="0.004"/>
  <testcase classname="testNGDemo.TestngDemo1" name="funA" time="0.01"/>
  <testcase classname="testNGDemo.TestngDemo2" name="funA" time="0.006"/>
  <testcase classname="testNGDemo.TestngDemo2" name="funB" time="0.012"/>
  <testcase classname="testNGDemo.TestngDemo2" name="funC" time="0.003"/>
  <testcase classname="testNGDemo.TestngDemo2" name="funD" time="0.002"/>
  <testcase classname="testNGDemo.TestngDemo3" name="funB" time="0.001"/>
  <testcase classname="testNGDemo.TestngDemo3" name="funC" time="0.002"/>
  <testcase classname="testNGDemo.TestngDemo3" name="funD" time="0.002"/>
  <testcase classname="testNGDemo.TestngDemo3" name="funA" time="0.002"/>
  <testcase classname="testNGDemo.TestngDemo4" name="loginTest" time="0.002">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at testNGDemo.TestngDemo4.loginTest(TestngDemo4.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="testNGDemo.TestngDemo4" name="customerDetails" time="0.0">
    <skipped/>
  </testcase> <!-- customerDetails -->
  <testcase classname="testNGDemo.TestngDemo5" name="funA" time="0.003"/>
  <testcase classname="testNGDemo.TestngDemo5" name="funB" time="0.002">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at testNGDemo.TestngDemo5.funB(TestngDemo5.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- funB -->
  <testcase classname="testNGDemo.TestngDemo5" name="funC" time="0.001">
    <failure type="java.lang.Exception">
      <![CDATA[java.lang.Exception
at testNGDemo.TestngDemo5.funC(TestngDemo5.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- funC -->
  <testcase classname="testNGDemo.TestngGroupsDemo1" name="funA" time="0.001"/>
  <testcase classname="testNGDemo.TestngGroupsDemo1" name="funB" time="0.0"/>
  <testcase classname="testNGDemo.TestngGroupsDemo1" name="funC" time="0.002"/>
  <testcase classname="testNGDemo.TestngGroupsDemo1" name="funD" time="0.002"/>
  <testcase classname="testNGDemo.TestngnnotationsDemo3" name="funA" time="0.001"/>
  <testcase classname="testNGDemo.TestngDemo5" name="funD" time="0.0">
    <ignored/>
  </testcase> <!-- funD -->
</testsuite> <!-- Regression suite 1 -->
